sets/spike/prime-core/hand-controlled-grabber/main.py

```diff
@@ -1,12 +1,3 @@
-"""
-This program is for Hand-Controlled Grabbers
-(in the "Invention Squad: Super Clean-Up" lesson unit).
-
-Follow the corresponding building instructions in the LEGO® SPIKE™ Prime App.
-
-Press the Force Sensor to grab objects, and release the Force Sensor to let go.
```

Comment:
I think it would be good to at least keep this one line description of the program in the code.

Reply:
@dlech sure, I've got @ArianaLuongPham to add back the description in the form of a comment (not docstring)